src/main/java/org/sagebionetworks/web/client/presenter/TeamPresenter.java

```diff
@@ -61,7 +61,6 @@ public class TeamPresenter
   private IsACTMemberAsyncHandler isACTMemberAsyncHandler;
   private TeamProjectsModalWidget teamProjectsModalWidget;
   private PortalGinInjector ginInjector;
-  private FeatureFlagConfig featureFlagConfig;
```

Comment:
Clean up member variable, which is not used (config is only referenced during construction)